service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageController.kt

```diff
@@ -49,7 +49,7 @@ class MessageController {
         user: AuthenticatedUser
     ): UnreadMessagesResponse {
         val accountIds = db.read { it.getMessageAccountsForUser(user) }.map { it.id }
-        val count = if (accountIds.isEmpty()) 0 else db.read { it.getUnreadMessages(accountIds) }
+        val count = if (accountIds.isEmpty()) 0 else db.read { it.getUnreadMessages(accountIds.toSet()) }
```

Comment:
maybe these should be named more like getUnreadMessages**Count**, otherwise it sounds like it returns a list of the actual messages